### PR TITLE
Describe how to test in CONTRIBUTING.mD

### PR DESCRIPTION
**Pull request by sunglim**

### CODE REVIEW

**Comment by sunglim** [comment]
@arv for review

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,7 +1,23 @@
...
 [individual](http://noms.io/ca_individual.html) or
 [corporation](http://noms.io/ca_corporation.html), before a
 pull request can be accepted.
+
+Running the tests
+-----------------
+
+You can use `go test` command, e.g:
+
+ - `go test ./cmd/...` should run every tests under the `cmd` directory.
```
Comment (arv) [inline CONTRIBUTING.md]: You can also do

```
go test $(go list ./... | grep -v /vendor/)
```

Comment (arv) [inline CONTRIBUTING.md]: There is also a bunch of js tests:

```
python tools/run-all-js-tests.py
```

**Comment by arv** [comment]
Thanks. This is very nice.

**Comment by mikegray** [comment]
I think this is already in the Hacking on Noms wiki.

File: CONTRIBUTING.md (hunk 1 above)
Comment (sunglim) [inline CONTRIBUTING.md]: Done.

**Comment by sunglim** [comment]
@arv  PTAL.

> I think this is already in the Hacking on Noms wiki.
@mikegray  It seems like we need to centralize them. My two cent is using `.md files` first. Just for   handling everything as source code.

**Comment by mikegray** [comment]
If we are moving this from Hacking on Noms wiki, it probably makes sense to move the other contents of that wiki to Contributing.md too. Can you please move those over too? Also just noticed the Apache License reference isn't a link - please link that to LICENSE at root of repo.

**Comment by aboodman** [comment]
yes. @sunglim - can you please copy the rest of the content from wiki page into this new contributing.md file.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,7 +1,27 @@
+Contributing to Noms
+====================
+
+License
+-------
+
 Noms is open source software, licensed under the [Apache License, Version 2.0]().
```
Comment (aboodman) [inline CONTRIBUTING.md]: Add a link to LICENSE ?

Comment (sunglim) [inline CONTRIBUTING.md]: Done.

**Comment by sunglim** [comment]
Sorry, I was late.

> If we are moving this from Hacking on Noms wiki, it probably makes sense to move the other contents of that wiki to Contributing.md too. Can you please move those over too? Also just noticed the Apache License reference isn't a link - please link that to LICENSE at root of repo.

Done. PTAL.

**Comment by arv** [comment]
LGTM

**Pull request merged**